Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelCancelEditingTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+final class SitterProfileViewModelCancelEditingTests: XCTestCase {
+    func testCancelEditingMethodExistsInAPI() {
+        SitterProfileViewModel().cancelEditing()
+    }
+
+    func testCancelEditingMethodSuccessfullyRestoresOriginalSitterProperties() {
```

Comment:
I didn't understand your intentions in this test. What do you test? Can you explain?